modules/core/client/app/config.js

```diff
@@ -1,23 +1,29 @@
-'use strict';
+(function (window) {
+  'use strict';
 
-// Init the application configuration module for AngularJS application
-var ApplicationConfiguration = (function () {
-  // Init module configuration options
-  var applicationModuleName = 'mean';
-  var applicationModuleVendorDependencies = ['ngResource', 'ngAnimate', 'ngMessages', 'ui.router', 'ui.bootstrap', 'ui.utils', 'angularFileUpload'];
+  // Init the application configuration module for AngularJS application
+  var ApplicationConfiguration = (function () {
```

Comment:
Why another IIFE?  can't you just set window.applicationConfiguration = service?